### PR TITLE
11.0 sync port

### PR DESCRIPTION
**Pull request by trojikman**

### CODE REVIEW

File: sync/models/sync_trigger_cron.py [hunk 1]
```diff
@@ -20,11 +21,9 @@ class SyncTriggerCron(models.Model):
         "ir.cron", delegate=True, required=True, ondelete="cascade"
     )
 
-    @api.model_create_multi
-    def create(self, vals_list):
-        for vals in vals_list:
-            vals.setdefault("name", vals.get("trigger_name", "Sync"))
-        return super(SyncTriggerCron, self).create(vals_list)
+    def create(self, vals):
```
Comment (yelizariev) [inline sync/models/sync_trigger_cron.py:25]: Always use ``@api.model`` for create method

https://odoo-source.com/?q=def%20create%5C(self%2C%20vals%5C)&i=nope&files=&excludeFiles=po%24%7Cpot%24%7Cyml%24%7Cyaml%24%7Ccss%24%7C%2Fstatic%2Flib%2F&repos=13.0

File: sync/models/sync_trigger_mixin.py [hunk 2]
```diff
@@ -43,12 +44,10 @@ class SyncTriggerMixinModelId(models.AbstractModel):
     _name = "sync.trigger.mixin.model_id"
     _description = "Mixing to fill model_id field"
 
-    @api.model_create_multi
-    def create(self, vals_list):
+    def create(self, vals):
```
Comment (yelizariev) [inline sync/models/sync_trigger_mixin.py:48]: ``@api.model``

File: sync/models/sync_trigger_mixin.py [hunk 3]
```diff
@@ -62,9 +61,7 @@ def default_get(self, fields):
         vals["state"] = "code"
         return vals
 
-    @api.model_create_multi
-    def create(self, vals_list):
-        records = super().create(vals_list)
-        for r in records:
-            r.code = r.get_code()
-        return records
+    def create(self, vals):
```
Comment (yelizariev) [inline sync/models/sync_trigger_mixin.py:66]: ``@api.model``

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by trojikman** [comment]
I approve to merge it now

**Pull request merged**